### ISSUE TITLE
How to fix alert "A job is running beyond the limit you have set up"

### ISSUE THREAD

**Issue by rulotec** [Myddleware/myddleware#509]
Hi, I'm getting this email every 30 min:

> Hi,
> 
> A job is running beyond the limit you have set up : 60 minutes.
> 
> The job has begun at 2020-07-15 06:00:06. ID of this job : 5f0e9b6606dac4.21814008
> 
> Please, check if this job is still transferring data. If it doesn't, please stop the job manually by opening the job and clicking on the button "Stop task".
> 
> Best regards,
> Myddleware team

Does "Job" refer to a "Transfer"? If so, how can I find it? I couldn't find a way to filter by `Doc Id`:

![image](https://user-images.githubusercontent.com/36896420/115093401-cda57380-9edf-11eb-91e8-32911c3fbc0f.png)

**Comment by stlgaits** [comment]
Hello,

You need to stop the task that is currently running. To do so, go to the 'Tasks' tab 
![image](https://user-images.githubusercontent.com/74190794/115208723-2aca4080-a0fd-11eb-94ad-c8238bfbc854.png)
select the correct task that has transfers in error and there you should have a button saying 'Stop Task' .
![image](https://user-images.githubusercontent.com/74190794/115208943-68c76480-a0fd-11eb-9586-8b72aa332dd0.png)

**Comment by rulotec** [comment]
Thanks for the tip!

However,I don't see tasks with errors. I have 50 pages with tasks and I searched all of them.

In the notification, there is an ID (5f0e9b6606dac4.21814008) but there's no way to search by ID. Even I tried by first going to another task and then replacing the ID in the URL, but there's no contents: `https://my-myddlware-instance/app.php/rule/task/view/5f0e9b6606dac4.21814008/log/`

Is there a chance that the notification is related to a task that's already been (logically) deleted?

**Comment by rulotec** [comment]
Also, I have two myddlewre instances. I searched in both of them. But from the notification, how do I know from which instance it is coming from?

**Comment by Myddleware** [comment]
Hi @rulotec ,
The open task are displayed in fisrt position so if you don't see it in the fist page, it is because there is an issue somewhere.
Could you check your Myddleware database by searching the task Id you have in  the table Job  ? If you find a task, we would be very interested in having a screenshot to try to reproduce the issue.

If you find it in your database, you can also change manually the status to 'End'. It should stop the email sent.

**Comment by rulotec** [comment]
Hi,

I finally found the problem. The message was coming from an unexpected myddleware instance. I have 4 in total and I just looked into the two ones I had recently installed. As Estelle, suggested, I could stop the task.

However, it would be really useful if those emails could come with a link that directly takes you to the task with the problem. That would make it a lot easier to debug.

Thanks!

**Comment by rulotec** [comment]
Also, it would be good if there was a way to turn notifications on/off or having the ability to set the interval.

**Comment by Myddleware** [comment]
Hi @rulotec,
Adding a link in the alert email is a gread idea.

If you want to turn notifications on/off , you can go to this page : 
![image](https://user-images.githubusercontent.com/11330301/128645800-82af68c4-bae9-40a9-ac7b-3f8847adaf49.png)

Hope it helps.

**Comment by rulotec** [comment]
Thanks! That's good to know.

**Comment by rulotec** [comment]
Hi guys!

Today  I wanted to see whether the email notifications were active for a particular instance. I came across the jobscheduler list:

![image](https://user-images.githubusercontent.com/36896420/136828065-85dd1860-1ad7-4fdd-9f2b-68774ef304b3.png)

However, I did not realize that the "Active" column was what I was looking for. I remembered you helped me with this a few months ago, so I came here and realized that I overlooked it.

I would like to make a suggestion here to change the name of the column to "Notifications active" instead of just "Active". The latter is ambiguous and makes you think that it refers to he cron job being active or something else.

In general I have found strings that are ambiguous or seem to be a literal translation from French that is hard to understand In english. For example "Rerun all errors" or "Delete the research filters".  

In the image below, for example, I would change Opened, Closed, Cancelled and Error to "Records in progress", "Records transfered", "Records cancelled" , "Records with errors":

![image](https://user-images.githubusercontent.com/36896420/136829153-fb44f887-5558-43b2-a0b3-c1dde38edc95.png)


I hope this feedback helps to improve the tool!
